src/modules/TeamsList/components/Teams/components/MyTeam/MyTeam.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC, useState } from 'react';
+import { StyledMyTeam, HeaderDecor, TableWrapper } from './styled';
+import { Button } from 'typography';
+import { DARK_TEXT_COLOR, WHITE_COLOR } from 'appConstants/colors';
+import { Team } from 'types';
+import { MembersListToggle } from '../MemberListToggle/MemberListToggle';
+import { MyTeamInfoBlock } from './components/MyTeamInfoBlock/MyTeamInfoBlock';
+import { TeamUserTable } from '../TeamUserTable/TeamUserTable';
+
+type MyTeamProps = {
+  team?: Team;
+};
+
+export const MyTeam: FC<MyTeamProps> = ({ team }) => {
+  const [isOpen, setOpenState] = useState(false);
+
+  const toggleListHandler = () => {
+    setOpenState(!isOpen);
+  };
+
+  const countMember: number | undefined = team?.members.length;
```

Comment:
you do not need to define types to variables like that.
const countMember = team?.members.length; - is enough.

Reply:
Ok, deleted